linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_tally2_file_selinux/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+title: 'An SELinux Context must be configured for default pam_tally2 file option'
+
+description: |-
+    The <tt>file</tt> configuration option in PAM pam_tally2.so module defines where to keep counts.
+    Default is /var/log/tallylog. The configured directory must have the correct SELinux context.
+
+rationale: |-
+    Not having the correct SELinux context on the pam_tally2.so file may lead to
+    unauthorized access to the directory.
+
+severity: medium
+
+identifiers:
+    cce@slmicro5: CCE-94088-2
+
+references:
+    disa: CCI-000044
+    nist: AC-7 (a)
+    srg: SRG-OS-000021-GPOS-00005
+
+platform: package[pam]
+
+ocil_clause: 'the security context type of the non-default tally directory is not "faillog_t"'
+
+ocil: |-
+    If the system does not have SELinux enabled and enforcing a targeted policy,
+    or if the pam_tally2 module is not configured for use, this requirement is not applicable
+
+    Check the security context type of the default tally2 directory with the following command:
+
+    $ sudo ls -Z /var/log/tallylog
+
+    unconfined_u:object_r:faillog_t:s0 /var/log/faillock
+
+    If the security context type of the tally directory is not "tallylog_t", this is a finding.
```

Comment:
should be faillog_t

Reply:
Thanks for the note , should be fixed in 0643f880be9586eec6b461a323e8b7f798c18546 :bow: